bluez-async/src/lib.rs

```diff
@@ -631,6 +634,19 @@ impl BluetoothSession {
         )
     }
 
+    fn device_with_timeout(&self, id: &DeviceId, timeout: Duration) -> impl OrgBluezDevice1 + Introspectable + Properties {
```

Comment:
Rather than making a separate method for it, I think it's better just to modify the existing `device` method to take a timeout, and pass the default `DBUS_METHOD_CALL_TIMEOUT` in places where it's not passed by the library user.

Reply:
Agree, fixed

Reply:
It doesn't look like you've done that.

Reply:
ok, fixed in both : device() + device_with_timeout() + connect() + connect_with_timeout()